hathorlib/utils.py

```diff
@@ -180,3 +183,15 @@ def get_hash160(public_key_bytes: bytes) -> bytes:
     h = hashlib.new('ripemd160')
     h.update(key_hash.digest())
     return h.digest()
+
+
+def is_nft_creation(tx: 'BaseTransaction') -> bool:
+    """Returns True if the transaction is an NFT creation"""
+    from hathorlib import TokenCreationTransaction
+    from hathorlib.base_transaction import TxVersion
+    if tx.version != TxVersion.TOKEN_CREATION_TRANSACTION:
+        return False
+
+    # Assert needed because of mypy strict
+    assert type(tx) == TokenCreationTransaction
```

Comment:
I think it'd be fine to meld this into the if above, and mypy will understand it, like this:

```python
def is_nft_creation(tx: 'BaseTransaction') -> bool:
    """Returns True if the transaction is an NFT creation"""
    from hathorlib import TokenCreationTransaction
    if isinstance(tx, TokenCreationTransaction):
        return tx.is_nft_creation()
    else:
        return False
```

It would be fine because we already check the version before instantiating a `TokenCreationTransaction`.

Reply:
That's actually pretty good, thanks.